graylog2-web-interface/src/views/components/Select.jsx

```diff
@@ -11,6 +14,20 @@ const MultiValueRemove = (props) => {
   );
 };
 
+const OverlayInner = ({ children, style }: {children: Node, style?: Object}) => React.Children.map(children,
+  child => React.cloneElement(child, { style: Object.assign({}, style, child.props.style) }));
+
+const menu = (selectRef) => {
+  const defaultMinWidth = 200;
+  const containerWidth = get(selectRef, 'current.select.controlRef.offsetWidth');
+  const width = containerWidth > defaultMinWidth ? containerWidth : defaultMinWidth;
+  return base => ({
+    ...base,
+    position: 'relative',
+    width,
```

Comment:
With this implementation we are defining the width without a unit. The `px` info gets added automatically. Do you prefer adding `px` manually with a template string, to be more explicit?

Reply:
I think being explicit with the width is a good thing `50` does not tell me what width it will be... is that `px`, `vw`, `em`, etc